pkg/controller/disruption/disruption_test.go

```diff
@@ -181,9 +187,8 @@ func newFakeDisruptionController() (*disruptionController, *pdbStates) {
 	dc.rsListerSynced = alwaysReady
 	dc.dListerSynced = alwaysReady
 	dc.ssListerSynced = alwaysReady
-	ctx := context.TODO()
```

Comment:
from where `ctx.Done()` is coming after removing this?

Reply:
It's coming from the test. See `newFakeDisruptionControllerWithTime`